<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="separator"/>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>语言</string>
     </property>
     <addaction name="action_chinese"/>
     <addaction name="action_english"/>
     <addaction name="action_japanese"/>
     <addaction name="action_korean"/>
     <addaction name="action_deutsch"/>
    </widget>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="action_english">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_japanese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>日本語</string>
   </property>
  </action>
  <action name="action_korean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>한글</string>
   </property>
  </action>
  <action name="action_deutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>最近</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
